Storage/FirebaseStorageUI/NSURL+SDWebImageFIRStorageLoader.h

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright (c) 2016 Google Inc.
```

Comment:
nit: Copyright should be current year (2019) for all files.

Reply:
Done